### PR TITLE
Fancy up the PDA UI

### LINKED ISSUE #1025: PDA improvements
- Uplink is visible to everyone
- When you toggle the flashlight the PDA light doesnt turn on visually

### PR DESCRIPTION
**Pull request by FL-OZ**
(mostly) (the flashlight thing is ehh. no sprite for it, the sprite i made didnt look good, and i while i think its a useful idea, im not sure its necessary atm)
closes #1025

### CODE REVIEW

File: Content.Client/GameObjects/Components/PDA/PDABoundUserInterface.cs [hunk 1]
```diff
@@ -108,18 +125,96 @@ protected override void UpdateState(BoundUserInterfaceState state)
...
+        /// <param name="x"></param>
+        /// <returns></returns>
+        public static Color GetWeightedColor(int x)
+        {
+            var weightedColor = Color.Gray;
+            if (x <= 0)
+            {
+                return weightedColor;
+            }
+            if (x <= 5)
```
Comment (ComicIronic) [inline Content.Client/GameObjects/Components/PDA/PDABoundUserInterface.cs:157]: Why not just make this `0 <= x && x <= 5` and take away the first conditional?

File: Content.Client/GameObjects/Components/PDA/PDABoundUserInterface.cs [hunk 2]
```diff
@@ -108,18 +125,96 @@ protected override void UpdateState(BoundUserInterfaceState state)
...
+                weightedColor = Color.Purple;
+            }
+
+            return weightedColor;
+        }
+
+        public static string GetWeightedColorString(int x)
+        {
+            var weightedColor = "gray";
+            if (x <= 0)
```
Comment (ComicIronic) [inline Content.Client/GameObjects/Components/PDA/PDABoundUserInterface.cs:180]: Same question.

**Comment by SweptWasTaken** [comment]
Tentatively reopening this PR because I think it's a good foundation and someone else could contribute to it if you're done with it.

**Comment by FL-OZ** [comment]
> Tentatively reopening this PR because I think it's had work put into it and someone else could contribute to it if you're done with it.

leave me alone, creep

File: Content.Client/GameObjects/Components/PDA/PDABoundUserInterface.cs (hunk 1 above)
Comment (FL-OZ) [reply Content.Client/GameObjects/Components/PDA/PDABoundUserInterface.cs:157]: that.. doesnt achieve the desired effect at all. resolved.

**Comment by FL-OZ** [comment]
@PJB3005 @Zumorica re-review please

**Pull request closed without merging**